packages/payments-service/src/core/server.ts

```diff
@@ -1,9 +1,10 @@
+import 'isomorphic-fetch'
 import express from 'express'
 import bodyParser from 'body-parser'
 import session from 'express-session'
```

Comment:
Oh also, I've removed session middleware from event-status-service - I also copied it from the payments-server package! But I can't see how it'll be needed as we're building stateless services